src/adapter.js

```diff
@@ -12,7 +12,51 @@ export const mediaTypes = [
 ];
 
 export function detect(source) {
-  return !!source.match(/"?swagger"?:\s*["']2\.0["']/g);
+  return !!(_.isString(source)
+    ? source.match(/"?swagger"?:\s*["']2\.0["']/g)
+    : source.swagger === '2.0');
+}
+
+// Test whether a key is a special Swagger extension.
+function isExtension(value, key) {
+  return key.indexOf('x-') === 0;
+}
+
+// Test whether tags can be treated as resource groups, and if so it sets a
+// group name for each resource (used later to create groups).
```

Comment:
@smizell @danielgtaylor I would consider Swagger Tags out of scope of the MVP. I would propose to throw a annotation with warning if there are tags present in the swagger object or in the operation object. Does this work for you guys?